src/input_adaptors/parser_adaptor.py

```diff
@@ -0,0 +1,17 @@
+import os
+from iadaptor import IInputAdaptor
+import sys
+sys.path.append(os.path.join(os.getcwd().replace("tests", '').replace("src", '').replace("input_adaptors", ''), "src", "output"))
+
+
+class ParsedInput(IInputAdaptor):
+    def __init__(self, parser):
+        self.command = str(parser.command).lower()
+        if self.command == "none" or self.command is None:
```

Comment:
How u lower case null variable 
You need check if parser.command == none
Throw